src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -265,6 +265,12 @@
     </Compile>
     <Compile Include="System\Data\SqlClient\SqlFileStream.Windows.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
+    <Compile Include="System\Data\SqlClient\SNI\SNIPacket.NetCoreApp.cs" />
```

Comment:
This Compile Include could be moved after `    <Compile Include="System\Data\SqlClient\SqlConnectionStringBuilder.NetCoreApp.cs" />` instead of creating a new `ItemGroup`